frontend/public/components/service.jsx

```diff
@@ -22,7 +22,7 @@ const menuActions = [Cog.factory.ModifyPodSelector, ...Cog.factory.common];
 const ServiceIP = ({s}) => {
   const children = _.map(s.spec.ports, (portObj, i) => {
     const clusterIP = s.spec.clusterIP === 'None' ? 'None' : `${s.spec.clusterIP}:${portObj.port}`;
-    return <span key={i}>{clusterIP}&nbsp;&nbsp;</span>;
+    return <div key={i}>{clusterIP}</div>;
```

Comment:
These get wrapped in a `<p>` tag on line 28, but divs aren't allowed in paragraphs. Either we should use `<span>` with `<br />`, or the p should be changed to a div. I'm not sure how either option would affect styling.